### ISSUE TITLE
Exclude labels in sync query

### ISSUE THREAD

**Issue by Mattrogers13** [omnivore-app/obsidian-omnivore#66]
Is it possible to exclude labels in a sync query? For example, **label:!health**. This didn't seem to work when tried.

**Comment by Mattrogers13** [comment]
I am a dolt, I just updated and read through the page https://omnivore.app/help/search.

**Comment by Mattrogers13** [comment]
using the query "-label:ttrpg" worked great, but I did received an "error fetching articles" message. The console revealed this:

```
YAMLParseError: Nested mappings are not allowed in compact mappings at line 2, column 8:

title: D&D: Take A Look At Matt Mercer's D&D Session Notes
       ^

    at Wg.onError (app.js:1:688494)
    at app.js:1:676625
    at Pg (app.js:1:677092)
    at Ug (app.js:1:687663)
    at app.js:1:688859
    at Wg.next [as push] (app.js:1:689001)
    at rv.pop (app.js:1:703188)
    at rv.end (app.js:1:702377)
    at rv.parse (app.js:1:702334)
    at app.js:1:712200
fetchOmnivore @ plugin:obsidian-omnivore:8190
```

```
YAMLParseError: Unexpected scalar at node end at line 2, column 8:

title: "dnd 5e - How do I decide what is a "reasonable" Suggestion? - Role-play…
       ^

    at Wg.onError (app.js:1:688494)
    at l (app.js:1:683498)
    at Lg (app.js:1:675665)
    at Og (app.js:1:684848)
    at Hg (app.js:1:685796)
    at Ug (app.js:1:687587)
    at app.js:1:676841
    at Pg (app.js:1:677092)
    at Ug (app.js:1:687663)
    at app.js:1:688859
fetchOmnivore @ plugin:obsidian-omnivore:8190
```

**Comment by sywhb** [comment]
Hey @Mattrogers13 , sorry for the delay. I have fixed this error in the latest release `1.3.1`.

It includes:
## Bug Fixes

* Fixed `Failed to fetch articles` error due to invalid YAML format in the front matter. Now you can define which variables to be used in the front matter

## Feature
* Added a reset template button to reset the template to default

![](https://cdn.discordapp.com/attachments/855181471643861002/1110401373855547443/Screenshot_2023-05-23_at_10.58.03_AM.png)

![](https://cdn.discordapp.com/attachments/855181471643861002/1110401374111408208/Screenshot_2023-05-23_at_10.58.19_AM.png)

Please let me know if it works for you and looking forward to your feedback

**Comment by ShaddyDC** [comment]
Unfortunately, `1.3.3` didn't work for me, and it just got stuck syncing forever.
Or rather, it didn't seem to be doing anything, but if I tried to sync again, it told me it was already syncing.
There was no output in the developer console besides the message that the sync started.
I tried this after resetting the template to the default.

Downgrading to `1.2.3` worked.

On a related note, can you update the default template under [this link](https://docs.omnivore.app/integrations/obsidian.html#controlling-the-layout-of-the-data-imported-to-obsidian)? Just to mention the multi line strings or something.
